plugins/captcha/math/services/provider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Captcha.math
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Extension\PluginInterface;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\DI\Container;
+use Joomla\DI\ServiceProviderInterface;
+use Joomla\Event\DispatcherInterface;
+use Joomla\Plugin\Captcha\Math\Extension\MathCaptcha;
+
+return new class () implements ServiceProviderInterface {
+    /**
+     * Registers the service provider with a DI container.
+     *
+     * @param   Container  $container  The DI container.
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    public function register(Container $container)
+    {
+        $container->set(
+            PluginInterface::class,
+            function (Container $container) {
+                $plugin = new MathCaptcha(
```

Comment:
```suggestion
                $plugin = new Math(
```
The name of the class differs from what is standard in Joomla. You probably have a good reason for that. 

This means that the following file also needs to be renamed:
plugins/captcha/**math**/src/Extension/**MathCaptcha.php** to plugins/captcha/**math**/src/Extension/**Math.php**

Reply:
Renamed